package.json

```diff
@@ -39,14 +39,14 @@
   },
   "scripts": {
     "build": "gulp",
-    "feature-tests": "./node_modules/.bin/cucumber-js test/feature --format=progress",
+    "feature-tests": "cucumber-js test/feature --format=progress",
```

Comment:
So that you can call them from your shell

npm runs these scripts in it's own environment